src/pages/patientView/genomicOverview/GenomicOverview.tsx

```diff
@@ -49,6 +51,8 @@ interface IGenomicOverviewProps {
     onSelectGenePanel?: (name: string) => void;
     disableTooltip?: boolean;
     store?: IGenomicOverviewStore;
+    mutationsStore?: PatientViewMutationsDataStore;
+    cnasStore?: PatientViewCnaDataStore;
```

Comment:
I would define a prop like `onResetView?:  () => void`, and handle the rest outside of this component. This component does not need to know anything about cna or mutation tables.